scripts/v8-to-v9.py

```diff
@@ -5,6 +5,16 @@
 import argparse
 from typing import Any
 
+def replace_in_obj(obj, old, new):
+    if isinstance(obj, dict):
+        return {k: replace_in_obj(v, old, new) for k, v in obj.items()}
+    elif isinstance(obj, list):
+        return [replace_in_obj(item, old, new) for item in obj]
+    elif isinstance(obj, str):
+        return obj.replace(old, new)
```

Comment:
string.replace is case sensitive, so should we consider convert the `obj`, `old` and `new` to lower case?